src/main/java/com/github/davidmoten/rx2/flowable/Transformers.java

```diff
@@ -4,7 +4,13 @@
 import java.util.Comparator;
 import java.util.List;
 import java.util.concurrent.Callable;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
 
+import com.github.davidmoten.guavamini.Optional;
+import com.github.davidmoten.rx2.internal.flowable.*;
```

Comment:
happy to leave this here for the moment but is not my practice to do this. Ctrl-Shift-O in Eclipse will replace these in an instant sorry!